Submissions/ottogugel.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "",
+  "name": "Otto Gugel",
+  "img": "",
+  "email":"",

```

Comment:
please provide an email - otherwise how would people contact you? (you can still leave this blank)